src/main/java/org/folio/oaipmh/helpers/RepositoryConfigurationHelper.java

```diff
@@ -70,7 +70,15 @@
             .forEach(o ->
               config.put(((JsonObject) o).getString("code"),
                 ((JsonObject) o).getString("value")));
+          // this should be removed once proper support of the config per tenant is added
```

Comment:
It is correct, however the approach is to store configs once they are retrieved to vert.x context so it can be accessible by main business logic. As far as I can see the context is shared between different calls to module that is why the configs are shared between all the requests

Reply:
got it.  I somehow missed that we were storing the configuration on the vertx context.  Does this have the side effect of using the "last known" configuration if mod-configuration is temporarily unreachable?

Reply:
Yes, once the configs are stored, they are available all the time in the context. The logic here is to use `merge in` approach which overrides old values by new ones. In case there are no configs retrieved from mod-configs or some config was deleted, the old configs (by key) will be still stored in context. To be honest I am not sure if someone is really going to remove completely configs to rely on system default values...